sbin/build.sh

```diff
@@ -877,6 +877,39 @@ generateSBoM() {
   addSBOMMetadataProperty "${javaHome}" "${classpath}" "${sbomJson}" "OS version" "${BUILD_CONFIG[OS_FULL_VERSION]^}"
   addSBOMMetadataProperty "${javaHome}" "${classpath}" "${sbomJson}" "OS architecture" "${BUILD_CONFIG[OS_ARCHITECTURE]^}"
 
+  # Create JDK Component
```

Comment:
this block looks like wrong. This was the original block which created a single component: JDK
However, later on for each component that is generated during the build, a separate SBOM component will be generated. So this block should be removed and the formulation should be added in the for loop for each component imho.

Reply:
Agreed - a side effect of attempting a rebase after your change went in - I'll need to do some manual fixup by the look of it. Thanks.

Reply:
OK should be resolved now.